src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -132,7 +132,8 @@ protected override async Task InitializeCoreAsync(CancellationToken cancellation
                 target: DataflowBlockFactory.CreateActionBlock<IProjectVersionedValue<(ConfiguredProject ActiveConfiguredProject, ConfigurationSubscriptionSources Sources)>>(
                     async update => await ExecuteUnderLockAsync(cancellationToken => OnSlicesChanged(update, cancellationToken)),
                     _unconfiguredProject,
-                    ProjectFaultSeverity.LimitedFunctionality),
+                    ProjectFaultSeverity.LimitedFunctionality,
+                    "LanguageServiceHostSlices {0}"),
```

Comment:
What fills in the place holder here, and what value will it have?

Reply:
I've only ever seen it used in WinDBG, to help identify the purpose behind each dataflow block in the graph. I'm not sure what value goes in the placeholder actually, or why it's needed, but all `nameFormat` values have that.

Reply:
Digging into this, the `nameFormat` is used on the dataflow block's `ToString` method, and is passed two placeholder values:
- `{0}` &mdash; the block's type name
- `{1}` &mdash; the hash code of the block